styleguide/_themes/derek/scss/components/_navigation.scss

```diff
@@ -1,110 +1,129 @@
-.nav > li > a:focus {
-  background-color: $signup-green;
+.nav > li > a {
+  &:focus {
+    background-color: $signup-green;
+  }
 }
 
 .main-nav {
-  border: 0;
   background: $brand-black;
-  -webkit-border-radius: 0;
-  -moz-border-radius: 0;
+  border: 0;
   border-radius: 0;
-  border: none;
 
   .navbar-nav > li > a {
-    color: #E0E0E0;
```

Comment:
someplaces we're defining new colors, some are in hex and some are lighten or darken? can we meet up to talk about the one source of truth on how colors are happening?